discovery-provider/alembic/versions/f775fb87f5ff_add_skipped_transaction_level_column.py

```diff
@@ -0,0 +1,37 @@
+"""add skipped_transaction level column
+
+Revision ID: f775fb87f5ff
+Revises: be27a2794f75
+Create Date: 2022-01-12 22:32:24.949547
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "f775fb87f5ff"
+down_revision = "be27a2794f75"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    skippedtransactionlevel = sa.Enum("node", "network", name="skippedtransactionlevel")
+    skippedtransactionlevel.create(op.get_bind())
+    op.add_column(
+        "skipped_transactions",
+        sa.Column(
+            "level",
+            sa.Enum("node", "network", name="skippedtransactionlevel"),
+            nullable=True,
+        ),
+    )
+    op.execute("UPDATE skipped_transactions SET level = 'network'")
```

Comment:
learned this technique [here](https://medium.com/the-andela-way/alembic-how-to-add-a-non-nullable-field-to-a-populated-table-998554003134)